### PR TITLE
Fix double exporting output pin

### PR DESCRIPTION
**Pull request by IonBazan**
Thanks to @joltmode for original solution.
I have modified it a bit to use `FileSystemInterface` and fixed tests.

### CODE REVIEW

**Comment by AndrewCarterUK** [comment]
Hey - I'm really sorry for missing this.

This looks like a really good fix - I'm holding fire on merging this for the moment as I don't have time to properly review.

I'm looking for someone to maintain this repository and if I can't find someone I'll probably mark the repository is inactive and then provide instructions for users on how to run this bugfix in the README.

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         }
     },
     "require": {
-        "php": "^7.3||^7.4||^8.0"
```
Comment (IonBazan) [inline composer.json:23]: `^7.3` already includes `^7.4`

**Comment by IonBazan** [comment]
@svpernova09 rebased 😄

**Comment by svpernova09** [comment]
Thanks for the PR!

**Pull request merged**